packages/mds-api-server/utils/tracer.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2021 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import tracer from 'dd-trace'
+tracer.init() // initialized in a different file to avoid hoisting.
```

Comment:
I think you need to retain the `enabled` option

Reply:
wooops!

Reply:
➕